mongodbatlas/data_source_mongodbatlas_data_lake_pipeline_run.go

```diff
@@ -0,0 +1,168 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const errorDataLakePipelineRunRead = "error reading MongoDB Atlas DataLake Run (%s): %s"
+
+func dataSourceMongoDBAtlasDataLakePipelineRun() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceMongoDBAtlasDataLakeRunRead,
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
```

Comment:
can we update this parameter to be pipeline_name to be consistent with Atlas Admin API docs if no concerns? https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Data-Lake-Pipelines/operation/getPipelineRun

also to better distinguish this parameter name from dataset_name parameter below

Reply:
updated